app/assets/javascripts/discourse/app/initializers/codeblock-buttons.js

```diff
@@ -0,0 +1,44 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import CodeblockButtons from "discourse/lib/codeblock-buttons";
+
+let _codeblockButtons = [];
```

Comment:
I think this should be automatically cleared on teardown. (to avoid cross-test leakage)

Reply:
I will take a look, do we have a teardown for initializers I guess?

Reply:
Oh, I realised I had this already, just have to clear the array :sweat_smile: